src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.20
+
+21 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### New Features
+
+- Add support for Openshift: Openshift Routes now show as Okteto Endpoints  <!-- 8025 -->
+- Enable Pull Secrets by default  <!-- 8001 -->
```

Comment:
Should we link to [here](https://www.okteto.com/docs/admin/registry-credentials/#pull-secrets-vs-kubelet-reconfiguration)?

Reply:
Also, I just remember that this is a new way of managing credentials to pull pods in the cluster and some customers already manage that in their own. We should include a note here or in another place (maybe adding a `before upgrade to 1.20.x` [here](https://www.okteto.com/docs/self-hosted/manage/upgrade/)) to mention that if someone doesn't want Okteto to manage them, they should disabled 2 settings:
* `regcredsManager.pullSecrets.enabled` should be set to `false`
* `daemonset.configurePrivateRegistriesInNodes.enabled` should be set to `false` (if they were already preventing Okteto to manage credentials in the node, they would probably have this one already set)